website/content/docs/enterprise/namespaces.mdx

```diff
@@ -7,11 +7,14 @@ description: Consul Enterprise enables data isolation with Namespaces.
 # Consul Enterprise Namespaces
 
 <EnterpriseAlert>
-  This feature requires{' '}
-  <a href="https://www.hashicorp.com/products/consul/">Consul Enterprise</a>{' '}.
+  This feature requires version 1.7.0+ of
+  HashiCorp Cloud Platform (HCP) or self-managed Consul Enterprise.
+  Refer to the{' '}
+  <a href="/docs/enterprise#consul-enterprise-feature-availability">enterprise feature matrix</a>
+  {' '}for additional information.
 </EnterpriseAlert>
 
-With Consul Enterprise v1.7.0, data for different users or teams
+With Consul Enterprise 1.7.0+, data for different users or teams
```

Comment:
```suggestion
With Consul Enterprise 1.7.0+, data for different users or teams
```
Do we need to repeat the version-specific information here? It's in the EnterpriseAlert box. I don't see that we're doing it in the following pages.